packages/documents/graphql/schema-types.js

```diff
@@ -8,12 +8,18 @@ type Series {
   episodes: [Episode!]!
 }
 
+type EpisodePart {
+  title: String
+  document: Document
+}
+
 type Episode {
   title: String
   label: String
   image: String
   publishDate: DateTime
   document: Document
+  parts: [EpisodePart]
```

Comment:
If not mistaken, you will return an empty parts array if there are no parts to it, right?

If so, schema is tighter with requiring a set to be returned `[...]!` (and returned entities are of `EpisodePart`, always, so `EpisodePart!`)

```suggestion
  parts: [EpisodePart!]!
```